omnigibson/prims/geom_prim.py

```diff
@@ -117,7 +117,7 @@ def opacity(self, opacity):
         if self.has_material():
             self.material.opacity_constant = opacity
         else:
-            self.set_attribute("primvars:displayOpacity", th.tensor([opacity]))
+            self.set_attribute("primvars:displayOpacity", np.array([opacity]))
```

Comment:
There is no `np` here and it's not allowed either. Does set_attribute not unwrap the tensor?

Reply:
nope, apparently np is required.